src/chromium/crNetworkManager.ts

```diff
@@ -91,17 +97,19 @@ export class CRNetworkManager {
   }
 
   async setCacheEnabled(enabled: boolean) {
```

Comment:
Why do we have a parameter which is now ignored?

Reply:
This is how it is declared in the delegate interface. 2 other browsers use that param. We can update that in a separate change.